### ISSUE TITLE
KBD Macro for to get VICE from bluescreen into Habitat...

### ISSUE THREAD

**Issue by frandallfarmer** [frandallfarmer/neohabitat#55]
All the developers would LOVE whoever can get a keyboard macro working to get us from starting VICE all the way into Habitat (assuming a disc that is already configured with an account, and starting the emulator with command line options that select that disc and install the symbol tables...)

Here's a variant of the key sequence I use to get from launching VICE:
[Alt-w] to start Warp Speed, if needed.
F1 (login screen)
F1 (select People Connection)
F7 (pull down menu in PC)
[Alt-w] (disable warp)
downarrow x 6 (to select Play a Game)
[Alt-w] (resume warp)
F1, F1, F1 (to select Club Caribe as the game)
[alt-8] (swap disc in drive 8)
clubcaribe1 + enter (select the first CC disc)    (sequence I use leverages file popdown: is c+down+enter)
[Enter] (to get it to open the app)
 --- slight pause to allow app to load and start
[alt-8] swap disk
clubcaribe2 + enter (second disc) (using popdown: c+down+down+enter)
[Enter] Tell Habitat to read the data disc.

DONE

**Comment by frandallfarmer** [comment]
Any chance we can automate this incase of crashes? This is the trickiest part of getting into Habitat - prone to many errors.

**Comment by smithrobs** [comment]
I made a super-flaky node script with the help of [robotjs](https://github.com/octalmage/robotjs). As long as everything goes right I can go from C64 start screen to ready-to-play local neohabitat in 25s with no clicking or typing.

I went down the road of a VICE snapshot recording, but it didn't seem to work with RS232 at all. We'd also have to work around (or wait for VICE to address) embedding the disk images in the snapshots.

I'm not sure of a better **cross-platform** solution. Re: automation of crashes, robotjs has some rudimentary image capture functionality but that seems like it would be a ton of work to get working right.

If you're interested I can clean it up, make it a bit more user-friendly, and send a PR for review.
